inc/class-camptix-payment-method.php

```diff
@@ -47,6 +47,22 @@ function __construct() {
 		}
 
 		$this->camptix_options = $camptix->get_options();
+		if (
+				array_key_exists( 'payment_methods', $this->camptix_options )
+				&& array_key_exists( $this->id, $this->camptix_options['payment_methods'] )
+				&& $this->camptix_options['payment_methods'][$this->id]
+		) {
+			add_filter( 'camptix_supported_currencies', array($this, 'add_supported_currency' ) );
+		}
+	}
+
+	/**
+	 * @param $currencies
+	 *
+	 * @return array Add currencies supported by this plugin to the global list of currencies
+	 */
+	function add_supported_currency( $currencies ) {
+		return array_merge( $this->supported_currencies, $currencies );
```

Comment:
Might be worth adding a code comment here to explain why it's `( $this->supported_currencies, $currencies )` and not `( $currencies, $this->supported_currencies )`